src/furo/__init__.py

```diff
@@ -283,7 +283,13 @@ def _builder_inited(app: sphinx.application.Sphinx) -> None:
     update_known_styles_state(app)
 
     def _update_default(key: str, *, new_default: Any) -> None:
-        app.config.values[key] = (new_default, *app.config.values[key][1:])
+        try:
+            conf_py_settings = app.config._raw_config
+        except AttributeError:
+            pass  # Sphinx's config has changed.
+        else:
+            if key not in conf_py_settings:
+                app.config.__dict__[key] = new_default
```

Comment:
This may be better (I think the current will ignore command-line overrides in Sphinx 7.3+):
```suggestion
                app.config._raw_config.setdefault(key, new_default)
```